owncloudApp/src/main/java/com/owncloud/android/ui/activity/WhatsNewActivity.java

```diff
@@ -177,28 +188,30 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
             mItem = getArguments() != null ? (FeatureItem) getArguments().getParcelable("feature") : null;
         }
 
+
         @Nullable
         @Override
         public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                                  @Nullable Bundle savedInstanceState) {
-            View v = inflater.inflate(R.layout.whats_new_element, container, false);
 
-            ImageView iv = v.findViewById(R.id.whatsNewImage);
+            WhatsNewElementBinding binding_element = WhatsNewElementBinding.inflate(getLayoutInflater());
```

Comment:
Just to polish it definitely, I would initialize this here but put the declaration around line 175, so that we have the binding variable available for all the methods of the fragment and not only for `onCreateView`:
```suggestion
            binding_element = WhatsNewElementBinding.inflate(getLayoutInflater());
```

And just before or after to the declaration of `mItem` (line 175):
```
private WhatsNewElementBinding binding_element;
```

Reply:
Hey @dev-divyansh, any news here? Just this change and I think we're ready to go 🚀